controllers/drplacementcontrol_controller_test.go

```diff
@@ -1009,4 +1170,170 @@ var _ = Describe("DRPlacementControl Reconciler", func() {
+				_, condition := getDRPCCondition(&drpc.Status, rmn.ConditionAvailable)
+				Expect(condition.Reason).To(Equal(string(rmn.FailedOver)))
+				userPlacementRule = getLatestUserPlacementRule(userPlacementRule.Name, userPlacementRule.Namespace)
+				Expect(userPlacementRule.Status.Decisions[0].ClusterName).To(Equal(East2ManagedCluster))
+			})
+		})
+		When("DRAction is cleared but DRPC status is empty", func() {
+			It("Should do nothing for placement as it is deployed somewhere else", func() {
+				// ----------------------------- Clear DRAction --------------------------------------
+				By("\n\n>>> Update DRPC status")
+				clearDRPCStatus()
+				waitForCompletion(string(""))
+				drpc = getLatestDRPC()
+				// At this point expect the DRPC status condition to have 2 types
+				// {Available and PeerReady}
+				// Final state didn't change and it is 'FailedOver' even though we tried to run
+				// initial deployment. But the status remains cleared.
+				Expect(drpc.Status.Phase).To(Equal(rmn.DRState("")))
+				userPlacementRule = getLatestUserPlacementRule(userPlacementRule.Name, userPlacementRule.Namespace)
+				Expect(userPlacementRule.Status.Decisions[0].ClusterName).To(Equal(East2ManagedCluster))
+			})
+		})
+		When("DRAction is set to Relocate", func() {
+			It("Should relocate to Primary (East1ManagedCluster)", func() {
+				// ----------------------------- RELOCATION TO PRIMARY FOR SYNC DR------------------------
+				By("\n\n*** relocate 2\n\n")
+				unfenceCluster(East1ManagedCluster)
+				relocateToPreferredCluster(userPlacementRule, East2ManagedCluster)
+				Expect(getManifestWorkCount(East1ManagedCluster)).Should(Equal(2)) // MWs for VRG+ROLES
+				Expect(getManifestWorkCount(East2ManagedCluster)).Should(Equal(1)) // Roles MW
+
+				drpc = getLatestDRPC()
+				// At this point expect the DRPC status condition to have 2 types
+				// {Available and PeerReady}
+				// Final state is 'Relocated'
+				Expect(drpc.Status.Phase).To(Equal(rmn.Relocated))
+				Expect(len(drpc.Status.Conditions)).To(Equal(2))
+				_, condition := getDRPCCondition(&drpc.Status, rmn.ConditionAvailable)
+				Expect(condition.Reason).To(Equal(string(rmn.Relocated)))
+				userPlacementRule = getLatestUserPlacementRule(userPlacementRule.Name, userPlacementRule.Namespace)
+				Expect(userPlacementRule.Status.Decisions[0].ClusterName).To(Equal(East1ManagedCluster))
+			})
+		})
+		When("Deleting user PlacementRule", func() {
+			It("Should cleanup DRPC", func() {
+				By("\n\n*** DELETE User PlacementRule ***\n\n")
+				deleteUserPlacementRule()
+				drpc := getLatestDRPC()
+				_, condition := getDRPCCondition(&drpc.Status, rmn.ConditionPeerReady)
+				Expect(condition).NotTo(BeNil())
+				// waitForCompletion("deleted")
+				// Expect(getManifestWorkCount(East1ManagedCluster)).Should(Equal(1)) // Roles MW
+			})
+		})
+
+		When("Deleting DRPC", func() {
+			It("Should delete VRG from Secondary (East2ManagedCluster)", func() {
```

Comment:
Should this be East1ManagedCluster? Because after relocate is done to East1ManagedCluster, VRG is expected to be there and for cleanup, it needs to be deleted from there.

In other words, it should be similar to what AsyncDR tests do.